tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -14,10 +14,9 @@ advanced/D#
                  kind: TYPE
                  properties: 4
                  display_name: "_"
-     self {
      declarations {
-+      symlinks: "advanced/D#_$2#"
```

Comment:
Considering this is a generated symbol and we don't sort the declarations I'm willing to accept this change.

Reply:
Likewise.